document/api/src/main/java/io/camunda/document/api/DocumentStoreRegistry.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.document.api;
+
+public interface DocumentStoreRegistry {
+
+  DocumentStoreRecord getDocumentStore(final String id);
+
+  DocumentStoreRecord getDefaultDocumentStore();
```

Comment:
We could also consider a sealed class that represents either the default store or a store with an idea. As the default store could be disabled. Might also want to consider an either here is well in case a store isnt found.

Reply:
Good point, I'm not sure at the moment if we will want to disable the default store (assumption is: one of the stores should be nominated as default anyway). But I added a todo to consider this later when we have more stores